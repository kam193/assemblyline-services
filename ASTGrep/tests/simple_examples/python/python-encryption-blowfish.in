Blowfish.new(b"12345678912345671234567891234567", Blowfish.MODE_CBC, iv=iv).decrypt(b'\xaaOM%iE\xf0\xda')
Blowfish.new(b"12345678912345671234567891234567", Blowfish.MODE_CBC, iv=b"12345678").decrypt(b'\xaaOM%iE\xf0\xda')
