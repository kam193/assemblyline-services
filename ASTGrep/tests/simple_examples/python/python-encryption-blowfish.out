Blowfish.new(b"12345678912345671234567891234567", Blowfish.MODE_CBC, iv=iv).decrypt(b'\xaaOM%iE\xf0\xda')
b'abcabcab'